README.md

```
@@ -0,0 +1,256 @@
+| 렉스 | `rex@mopl.io` | `1234` | USER |
+| 슬링키 | `slinky@mopl.io` | `1234` | USER |
+
+---
+
+## 👥 팀원 및 역할 (R&R)
+
+> 💻 **Backend Development Team** - 전원 백엔드 개발자로 구성된 팀입니다.
+
+<div align="center">
+
+| <img src="https://github.com/KarubiOhayo.png" width="120" height="120"/> | <img src="https://github.com/kjn4101.png" width="120" height="120"/> | <img src="https://github.com/chanhyeok0201.png" width="120" height="120"/> | <img src="https://github.com/rlawnsry.png" width="120" height="120"/> | <img src="https://github.com/woohyuk99.png" width="120" height="120"/> |
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)